Backend/src/main/java/com/example/backend/project/controller/ProjectController.java

```diff
@@ -42,18 +43,17 @@ public ResponseEntity getAllProjects(@RequestHeader String googleTokenEncoded){
 
     @PutMapping("/softLock/{projectId}")
```

Comment:
Rekli smo da prvo treba ici id pa onda softLock, tj. /{projectId]/softLock jer je to vise prema RESTful konotaciji. Isto tako treba to promijeniti u company i user controllerima.

Reply:
Hvala. Zaboravio.